ansible/roles/k8s-openebs-uninstall-helm/tasks/main.yaml

```diff
@@ -0,0 +1,100 @@
+---
+
+# Steps 
+
+#1) Find out the openebs chart name.
+#2) Uninstall openebs.
+#3) Verify if openebs is uninstalled succeffully.
+#4) Delete the namespace.
+#5) Uninstall helm.
+#6) Delete the resources.
+#7) Delete the test artifacts.
+
+- name: Getting OpenEBS chart name
+  shell: helm ls --all | grep openebs | grep DEPLOYED | awk '{print $1}'
```

Comment:
chartname = variable

Reply:
@yudaykiran  obtaining the chart name from deployment

Reply:
Obtaining the chart name based on the status and image name not the namespace name. So, it should be unique.

Reply:
@yudaykiran , we fetch the chart name from the list of charts deployed and then deleting it. Do you want to specify the chart name used in 'helm installation role' here also and attempt deleting it?